### PR TITLE
Release vhost-v0.12.0 and vhost-user-backend-v0.16.0 

### PR DESCRIPTION
**Pull request by russell-islam**
### Summary of the PR

*Please summarize here why the changes in this PR are needed.*

### Requirements

Before submitting your PR, please make sure you addressed the following
requirements:

- [ ] All commits in this PR have Signed-Off-By trailers (with
  `git commit -s`), and the commit message has max 60 characters for the
  summary and max 75 characters for each description line.
- [ ] All added/changed functionality has a corresponding unit/integration
  test.
- [ ] All added/changed public-facing functionality has entries in the "Upcoming 
  Release" section of CHANGELOG.md (if no such section exists, please create one).
- [ ] Any newly added `unsafe` code is properly documented.

### CODE REVIEW

**Comment by russell-islam** [comment]
@stefano-garzarella  Please take a look.

File: vhost/CHANGELOG.md [hunk 1]
```diff
@@ -1,20 +1,22 @@
 # Changelog
 ## [Unreleased]
+### Fixed
```
Comment (stefano-garzarella) [inline vhost/CHANGELOG.md:3]: Please prepare the unreleased section with all subsections, not just Fixed and Deprecated

File: vhost-user-backend/CHANGELOG.md [hunk 2]
```diff
@@ -1,17 +1,20 @@
 # Changelog
 ## [Unreleased]
 
+### Fixed
```
Comment (stefano-garzarella) [inline vhost-user-backend/CHANGELOG.md:4]: Ditto.

File: vhost-user-backend/Cargo.toml [hunk 3]
```diff
@@ -17,14 +17,14 @@ gpu-socket = ["vhost/gpu-socket"]
...
+vhost = { path = "../vhost", version = "0.12", features = ["vhost-user-backend"] }
 virtio-bindings = "0.2.1"
 virtio-queue = "0.13.0"
 vm-memory = { version = "0.15.0", features = ["backend-mmap", "backend-atomic", "backend-bitmap"] }
 vmm-sys-util = "0.12.1"
 
 [dev-dependencies]
 nix = { version = "0.29", features = ["fs"] }
-vhost = { path = "../vhost", version = "0.11", features = ["test-utils", "vhost-user-frontend", "vhost-user-backend"] }
+vhost = { path = "../vhost", version = "0.12", features = ["test-utils", "vhost-user-frontend", "vhost-user-backend"] }
```
Comment (stefano-garzarella) [inline vhost-user-backend/Cargo.toml:28]: These should be updated in the previous commit.

**Review by stefano-garzarella — COMMENTED** [review]
Thanks for that, just couple of comments.

File: vhost/CHANGELOG.md (hunk 1 above)
Comment (russell-islam) [reply vhost/CHANGELOG.md]: Thank you. I got it confused. It is now fixed.

**Review by stefano-garzarella — APPROVED** [review]

**Review by rbradford — APPROVED** [review]

**Comment by stefano-garzarella** [comment]
I'm going to create the releases and publish them.

**Comment by stefano-garzarella** [comment]
crates published:
https://crates.io/crates/vhost-user-backend/0.16.0
https://crates.io/crates/vhost/0.12.0

**Pull request merged**